blueocean-web/src/main/js/main.jsx

```diff
@@ -78,9 +78,13 @@ class App extends Component {
         });
 
         const topNavLinks = [
-            <Link query={location.query} to="/pipelines">{pipeCaption}</Link>,
+            <Extensions.Renderer extensionPoint="jenkins.blueocean.top.pipelines">
+                <Link query={location.query} to="/pipelines">{pipeCaption}</Link>
+            </Extensions.Renderer>,
             <Extensions.Renderer extensionPoint="jenkins.blueocean.top.links"/>,
-            <AdminLink t={translate} />,
+            <Extensions.Renderer extensionPoint="jenkins.blueocean.top.admin">
```

Comment:
There is one _very_ annoying thing about this, but nothing you can really do about it yet... the entire set of extensions is wrapped in a `<div>`, so you end up with a structure like:
```
<AdminLink>
<div>
    <AdminLink>
</div>
```

This can easily cause issues with styling if it's not taken into consideration